### PR TITLE
LLVM 9

### PR DESCRIPTION
**Pull request by manasij7479**

### CODE REVIEW

File: CMakeLists.txt [hunk 1]
```diff
@@ -38,7 +38,7 @@ execute_process(
   OUTPUT_VARIABLE LLVM_SYSTEM_LIBS
   OUTPUT_STRIP_TRAILING_WHITESPACE
 )
-set(LLVM_LIBS "${LLVM_LIBS} ${LLVM_SYSTEM_LIBS}")
+set(LLVM_LIBS "${LLVM_LIBS} ${LLVM_SYSTEM_LIBS} ${CMAKE_SOURCE_DIR}/third_party/llvm/${CMAKE_BUILD_TYPE}/lib/libclangASTMatchers.a")
```
Comment (regehr) [inline CMakeLists.txt]: this isn't the right solution, see how we set LLVM_LIBS in the first place and see if there's a flag that can be added there which gets the correct libraries

Comment (manasij7479) [reply CMakeLists.txt]: llvm-config does not seem to have a relevant flag

**Comment by regehr** [comment]
plz also bump z3 to latest release

File: CMakeLists.txt (hunk 1 above)
Comment (regehr) [reply CMakeLists.txt]: well, I assert that there is a solution to this which is different from bypassing llvm-config completely

File: lib/Infer/Pruning.cpp [hunk 2]
```diff
@@ -14,7 +14,7 @@
 
 #include "souper/Infer/AbstractInterpreter.h"
 #include "souper/Infer/Pruning.h"
-
+#include "souper/Extractor/Candidates.h"
```
Comment (regehr) [inline lib/Infer/Pruning.cpp]: what necessitated this change?

Comment (manasij7479) [reply lib/Infer/Pruning.cpp]: Added a getSetSize declaration in that header.

**Pull request closed without merging**